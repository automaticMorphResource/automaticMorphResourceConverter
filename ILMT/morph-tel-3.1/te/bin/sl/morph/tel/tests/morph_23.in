1	adilEd	unk	
1	nagarAniki	unk	
1	xAxApu	unk	
1	100	unk	
1	kilomItarla	unk	
1	xUraMlo	unk	
1	uMde	unk	
1	"	unk	
1	kep	unk	
1	jarvis	unk	
1	"	unk	
1	ane	unk	
1	revu	unk	
1	nuMdi	unk	
1	samuxraMlo	unk	
1	xAxApu	unk	
1	15	unk	
1	kilomItarla	unk	
1	mera	unk	
1	ixi	unk	
1	uMtuMxi	unk	
1	.	unk	
