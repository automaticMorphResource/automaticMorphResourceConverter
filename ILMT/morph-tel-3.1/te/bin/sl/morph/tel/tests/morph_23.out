1	adilEd	unk	<fs af='adilEd,n,,sg,,d,0,0'>|<fs af='adilEd,n,,sg,,o,ti,ti'>
2	nagarAniki	unk	<fs af='nagaraM,n,,sg,,,ki,ki'>
3	xAxApu	unk	<fs af='xAxApu,n,,sg,,d,0,0'>|<fs af='xAxApu,n,,sg,,o,ti,ti'>
4	100	unk	<fs af='100,num,,,,,,'>
5	kilomItarla	unk	<fs af='kilomItar,n,,pl,,o,ti,ti'>
6	xUraMlo	unk	<fs af='xUraM,n,,sg,,,lo,lo'>
7	uMde	unk	<fs af='uMdu,v,any,any,any,,e,e'>|<fs af='uMdu,v,any,any,any,,i,i_e'>
8	"	unk	<fs af='&bdquo;,punc,,,,,,'>
9	kep	unk	<fs af='kep,unk,,,,,,'>
10	jarvis	unk	<fs af='jarvis,n,,sg,,d,0,0'>|<fs af='jarvis,n,,sg,,o,ti,ti'>
11	"	unk	<fs af='&bdquo;,punc,,,,,,'>
12	ane	unk	<fs af='ane,avy,,,,,,'>|<fs af='ani,avy,,,,,e,e_avy'>|<fs af='ani,avy,,,,,e,e_avy'>|<fs af='anu,v,any,any,any,,e,e'>|<fs af='anu,v,any,any,any,,i,i_e'>
13	revu	unk	<fs af='revu,n,,sg,,d,0,0'>|<fs af='revu,n,,sg,,o,ti,ti'>|<fs af='re,n,,sg,,,vu,vu'>
14	nuMdi	unk	<fs af='nuMdi,avy,,,,,0,0_avy'>
15	samuxraMlo	unk	<fs af='samuxraM,n,,sg,,,lo,lo'>
16	xAxApu	unk	<fs af='xAxApu,n,,sg,,d,0,0'>|<fs af='xAxApu,n,,sg,,o,ti,ti'>
17	15	unk	<fs af='15,num,,,,,,'>
18	kilomItarla	unk	<fs af='kilomItar,n,,pl,,o,ti,ti'>
19	mera	unk	<fs af='mera,n,,sg,,d,0,0'>|<fs af='mera,n,,sg,,o,ti,ti'>
20	ixi	unk	<fs af='ixi,pn,fn,sg,3,d,0,0'>
21	uMtuMxi	unk	<fs af='uMdu,v,fn,sg,3,,wA,wA'>
22	.	unk	<fs af='&dot;,punc,,,,,,'>
